src/router/index.js

```diff
@@ -23,7 +23,7 @@ const routes = [
     // route level code-splitting
     // this generates a separate chunk (about.[hash].js) for this route
     // which is lazy-loaded when the route is visited.
-    component: () => import(/* webpackChunkName: "about" */ '../views/About.vue'),
```

Comment:
würde ich drin lassen, da dann der compiler das in unterschiedliche Dateien aufteilt und das Laden schneller wird: About guckt man ja nicht immer jedes mal an.

Reply:
Ich weiß gar nicht warum das rausgenommen wurde... aber ja natürlich das muss rein.